medical-portal/src/UI/src/app/dashboard/dashboard.component.ts

```diff
@@ -123,31 +123,28 @@ export class DashboardComponent {
 
   searchDmerCase(): void {
     console.log('search DMER Case');
-    if (
-      this.prevSearchBox === '' ||
-      this.prevSearchBox !== this.searchBox.value
-    ) {
-      let searchParams: Parameters<
-        CasesService['apiCasesSearchIdCodeGet$Json']
-      >[0] = {
-        idCode: this.searchBox.value as string,
-      };
-
-      this.isSearching = true;
-      this.noResults = false;
-
-      this.casesService.apiCasesSearchIdCodeGet$Json(searchParams).subscribe({
-        next: (dmerCase) => {
-          if (dmerCase) this.searchedCase = dmerCase;
-        },
-        error: (err) => {
-          this.noResults = true;
-        },
-        complete: () => {
-          this.isSearching = false;
-        },
-      });
-    }
+
+    let searchParams: Parameters<
+      CasesService['apiCasesSearchIdCodeGet$Json']
+    >[0] = {
+      idCode: this.searchBox.value as string,
+    };
+
+    this.isSearching = true;
+    this.noResults = false;
+
+    this.casesService.apiCasesSearchIdCodeGet$Json(searchParams).subscribe({
+      next: (dmerCase) => {
+        if (dmerCase) this.searchedCase = dmerCase;
+      },
+      error: (err) => {
+        this.noResults = true;
```

Comment:
I'd be tempted to log the error. We don't really have a standard yet for FE logging

Reply:
The error message is displayed in the HTML when noresults=true